bugs/gh-275.txt

```diff
@@ -0,0 +1,736 @@
+=== Begin SRU Template ===
+[Impact]
+On s390x legacy terminals the use of square brackets [] is not available, therefore
+that makes it harder for those terminal to pass cloudinit configuration through kernel cmdline.
+Therefore, we are now supporting %-escaped characters, allowing those terminal to pass more complex
+config through kernel cmdline.
+
+[Test Case]
+
+```
+#!/bin/sh
+set -x
+# Manually deploy on a lxc container
+
+CMD_ARGUMENT="cc: runcmd%3A%20%5B%20echo%20testkernelcmdline%20%5D end_cc"
+
+restart_cloudinit() {
```

Comment:
Good consolidated function. I'd like to start moving common util functions like this into sru-scripts/sru-vars.template so cloud tests can use this too.   Once we land #113 . It'll cut down on the size of the test scripts we have to write. Though, we might find other ways to shave time/cost off our SRU verification tests to. So, let's talk in SRU retrospective about this.